### PR TITLE
Write snapshot artifacts to disk ready for xcpretty

### PR DESCRIPTION
**Pull request by TomHumphrey150**

### CODE REVIEW

File: Sources/SnapshotTesting/Snapshotting/UIImage.swift [hunk 1]
```diff
@@ -14,21 +14,30 @@ extension Diffing where Value == UIImage {
...
         : "Newly-taken snapshot@\(new.size) does not match reference@\(old.size)."
-      let oldAttachment = XCTAttachment(image: old)
-      oldAttachment.name = "reference"
-      let newAttachment = XCTAttachment(image: new)
-      newAttachment.name = "failure"
-      let differenceAttachment = XCTAttachment(image: difference)
-      differenceAttachment.name = "difference"
-      return (
+        let oldArtifact = SnapshotArtifact(
+            data: old.pngData()!,
```
Comment (julia-ponomareva-cm) [inline Sources/SnapshotTesting/Snapshotting/UIImage.swift:24]: do we need force unwrap here? (and same for lines below)

**Review by julia-ponomareva-cm — COMMENTED** [review]
Don't have good context on this, but are we going to delete all of this artefacts at some point? (probably handled by library itself)?

File: Sources/SnapshotTesting/Snapshotting/UIImage.swift (hunk 1 above)
Comment (TomHumphrey150) [reply Sources/SnapshotTesting/Snapshotting/UIImage.swift:24]: This is taken from the original PR on the library, approved by the maintainers. Given that this is a test, I would say the force unwrap is worth it 🧑‍🎓

**Comment by TomHumphrey150** [comment]
> Don't have good context on this, but are we going to delete all of this artefacts at some point? (probably handled by library itself)?

My makefile automation clears the artifacts before running the tests, if that's what you mean? They can only be written 1x, after that they are just overwriting the same point on disk.

**Review by julia-ponomareva-cm — APPROVED** [review]

**Pull request merged**